FindMKL.cmake

```diff
@@ -1,52 +1,114 @@
+# Copyright (c) 2018-2019, ETH Zurich
+# BSD 3-Clause License. All rights reserved.
+#
+# Author: Teodor Nikolov (teodor.nikolov22@gmail.com)
```

Comment:
The project name is missing and the author is not needed.
```
#
# Cmake recipes
# https://github.com/eth-cscs/cmake-recipes
#
# Copyright (c) 2018-2019, ETH Zurich
# BSD 3-Clause License. All rights reserved.
#
```

Reply:
I would like to keep the author in case someone wants to contact me for issues.